Common/Source/Devices/devLX_EOS_ERA.cpp

```diff
@@ -405,6 +405,7 @@ int PortNum = d->PortNumber;
     dfe->addEnumText(MsgToken(491)); // LKTOKEN  _@M491_ "OFF"
     dfe->addEnumText(MsgToken(2452)); // LKTOKEN  _@M2452_ "IN"
     dfe->addEnumText(MsgToken(2453)); // LKTOKEN  _@M2453_ "OUT"
+    dfe->addEnumText(MsgToken(2454)); // LKTOKEN  _@M2454_ "IN & OUT"
```

Comment:
unrelated to change in "DevLX_EOS_ERA::PutTarget"

Reply:
right! should be related to STF switch!

Reply:
that is correct as it is it is intendend to have the possibility to trigger STF from the device and LK.